recipes/rp2biosensor/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "rp2biosensor" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/brsynth/rp2biosensor/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 4c620aafc0612de7af6b0c60a2c8273effb936601501f8c94d612c13a86947ce
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install -vv .
+
+requirements:
+  host:
+    - python >=3.7,<3.9
+    - pip
+  run:
+    - python >=3.7,<3.9
+    - networkx
+    - rdkit
+    - rr_cache
+
+test:
+  source_files:
+    - tests
+  imports:
+    - rp2biosensor
+  requires:
+    - pytest
+  commands:
+    - python -m rp2biosensor --help
```

Comment:
you'll want to add `pip` to requires and run `pip check` as well. This helps ensure that your upstreams are _also_ up-to-date with their metadata.

Reply:
also: if you're going to make an actual command line tool, in `setup.py` go ahead and make an `entry_points={"console_scripts": ["rp2biosensor = rp2biosensor:main"]}` or something along those lines. That needs to get hoisted in the `build` section of _this_ file: https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#python-entry-points

This helps ensure that all of the plumbing is set up properly to make cross-platform scripts, correctly added to the path, etc.

Reply:
Hi @bollwyvl ,

Thanks for your suggestions.

Sorry but we don't have PyPI packages, only sources from the github.

We could make some changes:
- add entry_points to the recipe and setup.py
- add pip check to the recipe
- include license.md to MANIFEST.in

Thank your for you work.

Best wishes.